CONTRIBUTING.md

```diff
@@ -59,43 +62,44 @@ Some examples:
 - `mysql {{database_name}}`
 - `unrar x {{compressed.rar}}`
 
-In short, make it as intuitive as possible for the user to figure out
-how to use the command and fill it in with values.
+In short, make it as intuitive as possible
```

Comment:
_if_ we are going for semantic-ish linebreaks then it should be split at the comma and the rest left as a single line.

That'd minimize diffs in the future, and it'd still be rendered just fine.